src/main/java/com/gluonhq/NativeBaseMojo.java

```diff
@@ -134,7 +134,9 @@ private void configSubstrate() {
         if (javaStaticSdkVersion != null) {
             clientConfig.setJavaStaticSdkVersion(javaStaticSdkVersion);
         }
-        clientConfig.setJavafxStaticSdkVersion(javafxStaticSdkVersion);
+        if (javafxStaticSdkVersion != null) {
```

Comment:
in both cases, check for `null` is redundant. Substrate uses default version if those are `null`

Reply:
It was fixed in my branch, with this commit https://github.com/jperedadnr/client-maven-plugin/commit/d3665660bc473f807a16c1eddc90af4d39f90ac5, but I don't know why it didn't make it to this open PR??

Reply:
Strange... looks like Github is having issues? Try to close this PR and create a new one maybe?

Reply:
ok, I'll do that